staging/src/k8s.io/apiserver/pkg/util/flowcontrol/conc_alloc.go

```diff
@@ -0,0 +1,252 @@
+/*
```

Comment:
I added couple comments about the solver into: https://github.com/kubernetes/kubernetes/pull/112137#pullrequestreview-1170103370

Reply:
The force-push to 7c06996bb9d picks up my response to that review.

Reply:
That PR has been already merged.
@MikeSpreitzer - can you please rebase?

Reply:
done